core/src/main/kotlin/io/customer/sdk/data/store/GlobalPreferenceStore.kt

```diff
@@ -23,6 +26,8 @@ internal class GlobalPreferenceStoreImpl(
     context: Context
 ) : PreferenceStore(context), GlobalPreferenceStore {
 
+    val scope: CoroutineScope = SDKComponent.scopeProvider.globalPreferenceStoreScope
```

Comment:
Could have been `private`?

```suggestion
    private val scope: CoroutineScope = SDKComponent.scopeProvider.globalPreferenceStoreScope
```